src/state.ml

```diff
@@ -264,24 +266,22 @@ let mtu config compress =
     | None -> 1500 (* TODO "client_merge_server_config" should do this! *)
     | Some x -> x
   in
-  let bs = match Config.find Cipher config with
-    | Some "AES-256-CBC" -> 16
-    | _ -> assert false
-  in
   (* padding, done on packet_id + compress + data *)
   let not_yet_padded_payload =
     4 (* packet id *) + if compress then 1 else 0
   in
   let hdrs =
-    3 (* hdr: 2 byte length, 1 byte op + key *) + bs (* IV *) + Packet.hmac_len
+    3 (* hdr: 2 byte length, 1 byte op + key *) +
+    Packet.block_size (* IV *) +
+    Packet.hmac_len
```

Comment:
For AES-256-GCM, which is now the default upstream, I believe `block size <> iv size` (12 bytes instead of 16)?

Reply:
maybe. out of scope -- let's fix issues when we encounter them (i.e. when someone implements multi-cipher support / AES-256-GCM).